src/lib/sw_engine/tvgSwFill.cpp

```diff
@@ -210,12 +209,11 @@ void fillFetchRadial(const SwFill* fill, uint32_t* dst, uint32_t y, uint32_t x,
     //Rotation
     auto rx = (x + 0.5f - fill->radial.cx) * fill->sy;
     auto ry = (y + 0.5f - fill->radial.cy) * fill->sx;
-    auto inv2a = fill->radial.inv2a;
     auto rxy = rx * rx + ry * ry;
     auto rxryPlus = 2 * rx;
-    auto det = (-4 * fill->radial.a * -rxy) * inv2a;
-    auto detDelta = (4 * fill->radial.a * (rxryPlus + 1.0f)) * inv2a;
-    auto detDelta2 = (4 * fill->radial.a * 2.0f) * inv2a;
+    auto det = rxy / fill->radial.a;
+    auto detDelta = (rxryPlus + 1.0f) / fill->radial.a;
+    auto detDelta2 = 2.0f / fill->radial.a;
```

Comment:
In general, Division is slower than multiplication.
What benefits can we get from this refactoring?

Reply:
I hanged to multiplication. 

- i think the code is more readable without this variables
- we have one float less in the SwRadial  struct
- in total we have only 3 multiplications instead of 9:
    * 1 instead of 3 in 'det'
    *  1 instead of 3 in 'detDelta'
    *  1 instead of 3 'detDelat2'

do you think it's not worth it?

Reply:
not to mention that we have now 2 LOC less ;)

Reply:
@mgrudzinska Here question is which function is more serious.  sometimes we can give up readibility for performance.
so..
feching is called per pixels vs prepare is called just one time.
let's think.

Reply:
@hermet you are very right, sorry
so now the benefits are no 'pow' called and less multiplications. it's still something